src/main/resources/static/assets/js/sdh.js

```diff
@@ -42,19 +71,35 @@
 					return '';
 				}
 		    },
-		    isShowDetails(field) {
+		    _isShowDetails(field) {
 		    	return field.key === 'show_details';
+		    },
+		    _formatDate(dateString) {
+		    	return dateString.replace(/ \d\d:\d\d:\d\d\.\d/, '');
 		    }
 		},
 		computed: {
+			clinicianReviewSummary () {				
+				return this.domainOptions.map( domain => {
+					let reviewForDomain = this.reviews.filter( review => {
+						return review.domain === domain.description;
+					});
+					if (reviewForDomain.length == 0) {
+						return {'domain': domain.description, 'date_last_reviewed': 'NA', 'clinician_priority':'None', 'patient_readiness': 'None'};
```

Comment:
Object keys don't need to be quoted unless they have characters other than letters, numbers, and underscore (e.g. spaces, minus sign).